content/en/logs/log_collection/javascript.md

```diff
@@ -103,7 +103,7 @@ The following parameters can be used to configure the library to send logs to Da
 
       // OPTIONAL
       // add global metadata attributes
```

Comment:
As we now can only add one attribute per call of the method let's remove the attribute`s` and say:

`add global metadata attribute, one attribute can be added at a time` or something similar.